packages/algolia/src/client.ts

```diff
@@ -88,6 +91,7 @@ export class AlgoliaSearchClient {
     entityTypes: T[],
     query: string,
     requestOptions?: SearchOptions,
+    descending?: boolean,
```

Comment:
I think this should be a parameter like sort as you cannot have two sort methods with algolia